sdks/python/apache_beam/yaml/yaml_provider.py

```diff
@@ -48,7 +48,8 @@
 from apache_beam.portability.api import schema_pb2
 from apache_beam.transforms import external
 from apache_beam.transforms import window
-from apache_beam.transforms.fully_qualified_named_transform import FullyQualifiedNamedTransform
+from apache_beam.transforms.fully_qualified_named_transform import \
+  FullyQualifiedNamedTransform
```

Comment:
Imports are an exception to line length limits; keep it on the same line. 

Also, don't use backslashes for line continuations: https://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
The IDE keeps auto-formatting that. I'll figure out how to disable that